src/pages/home.js

```diff
@@ -108,17 +114,37 @@ export function Home() {
           fetchBenefitsFailed
             ? fetchBenefitsFailedObj
             : fetchBenefitsCountFailedObj
```

Comment:
why is this being changed

Reply:
the lifejourneys were removed from main

Reply:
As part of another task not this one. PR's should be limited according to what you are working on